stream_alert_cli/package.py

```diff
@@ -19,9 +19,9 @@
 import base64
 import hashlib
 import os
+import pip
```

Comment:
I believe `pip` is a non-standard package (doesn't come with Python), so it would actually go after `boto3.` Not sure though